crates/workspace/src/pane.rs

```diff
@@ -2190,7 +2195,9 @@ impl Pane {
                             .read(cx)
                             .item_for_entry(entry, cx)
                             .and_then(|item| item.project_path(cx))
-                            .map(|project_path| project_path.path);
+                            .map(|project_path| project_path.path)
+                            .map_or(None, |path| if path.exists() { Some(path) } else { None });
+                        let has_relative_path = relative_path.is_some();
```

Comment:
This is also sus. `when_some()` call requires moving value, but the same value is used to check in two different places. I don't know what would be an idiomatic way. Calling `clone()` would be fine, I guess? But for this check I need a value.